test/migrate/fixtures/w-body-attr-val/snapshot-expected.marko

```diff
@@ -0,0 +1,8 @@
+<div>
+  <if(typeof input.renderBody === "string")>
```

Comment:
The change recommended above should fix this `<if(` which should be checking `typeof input.test.renderBody`.